### ISSUE TITLE
Doesn't work for brand new project

### ISSUE THREAD

**Issue by ffMathy** [SteveGilham/altcover#113]
Repro steps:
1. Make a C# Console application.
2. Make a MsTest test project that references the application.
3. Install `AltCover` on the test project.
4. Run `dotnet test` with some more advanced arguments (haven't tested with just simple parameters).

Error:
```
C:\Users\mathi\.nuget\packages\altcover\7.4.807\build\netstandard2.0\AltCover.targets(133,5): error MSB4044: The "AltCover.ContingentCopy" task was not given a value for the required parameter "FileName". [C:\Users\mathi\source\repos\ConsoleApp14\UnitTestProject1\UnitTestProject1.csproj]
C:\Users\mathi\.nuget\packages\altcover\7.4.807\build\netstandard2.0\AltCover.targets(179,5): warning : A total of 0 visits recorded [C:\Users\mathi\source\repos\ConsoleApp14\UnitTestProject1\UnitTestProject1.csproj]
```

Attached is the project.

[ConsoleApp14.zip](https://github.com/SteveGilham/altcover/files/5808689/ConsoleApp14.zip)

**Comment by SteveGilham** [comment]
Interesting.   That would me a `<None>` entry with an empty file path. Doesn't repro for the case
```
 "C:\Program Files\dotnet\dotnet.EXE"  test /p:AltCover=true /p:AltCoverAssemblyFilter=xunit /p:AltCoverReportFormat=OpenCover /p:AltCoverShowStatic=- /p:AltCoverShowSummary=Green /p:AltCoverForce=true /p:AltCoverImportModule=true /p:AltCoverGetVersion=true --verbosity minimal "" --configuration Debug /nodeReuse:False /p:CheckEolTargetFramework=false
```

Easy enough to fix, though.

**Comment by ffMathy** [comment]
Did you try on the attached file that contains the source code? Note that it's an MsTest project too, not an XUnit project.

**Comment by SteveGilham** [comment]
Yes -- I expanded that .zip archive, removed all the dot-folders and the TestResults folder, and wired the solution + 2 projects into an existing operational test as-is, hence the futile exclusions -- not that those should matter, the point being "did it raise an MSBuild error or not".

**Comment by ffMathy** [comment]
Ah - gotcha. That makes sense :+1:

**Comment by SteveGilham** [comment]
So, here's a build with a mitigation in -- allow the copy task argument to be unset, and handle that eventuality --
[altcover.7.4.140-github-pre.nupkg.zip](https://github.com/SteveGilham/altcover/files/5809253/altcover.7.4.140-github-pre.nupkg.zip)

**Comment by ffMathy** [comment]
That works! Thank you! :pray:

**Comment by SteveGilham** [comment]
Should be resolved in [v7.4.808](https://www.nuget.org/packages/AltCover/7.4.808)
